monai/transforms/intensity/dictionary.py

```diff
@@ -164,7 +168,7 @@ def randomize(self, im_shape: Sequence[int]) -> None:
         for m in self.mean:
             self._noise.append(self.R.normal(m, self.R.uniform(0, self.std), size=im_shape))
 
-    def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+    def __call__(self, data: DataObjects.Mapping) -> DataObjects.Dict:
```

Comment:
Should we simplify to only use `DataObjects.Dict`?
@wyli I don't quite remember why we don't use `Dict` in dict transforms?

Thanks.